lottery/src/lib.rs

```diff
@@ -14,6 +14,7 @@ use ft_io::*;
 use gstd::{debug, exec, msg, prelude::*, ActorId};
 use lt_io::*;
 use scale_info::TypeInfo;
+use sp_core::hashing::blake2_256;
```

Comment:
Please follow the idiomatic `use`.

https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths

```suggestion
use sp_core::hashing;
```

Reply:
Fixed